### ISSUE TITLE
Loading

### ISSUE THREAD

**Issue by marcelipe** [terjanq/xDiep.io#10]
Hi terjanq, I'm trying to use this script but it just stays loading forever. Is it still working?

**Comment by hxxr** [comment]
Did you make sure diep.io works without the script? I used to have this problem long before I even thought about scripts for diep.io.

**Comment by marcelipe** [comment]
Yep. When I turn Tampermonkey off diep works normally, when I turn it back on, doesn't load :'(

**Comment by marcelipe** [comment]
Just oppened Tampermonkey and got a error message.
![screenshot_1](https://cloud.githubusercontent.com/assets/22745765/19236893/33eaa05e-8ed0-11e6-9eb2-2379ea9d0749.png)

**Comment by terjanq** [comment]
Hi, sadly my script doesn't work anymore. It needs some improvements to work again

**Comment by marcelipe** [comment]
@terjanq no problem. Thx anyway

**Comment by Windfall-Github** [comment]
Hey @terjanq ,
I posted a new issue, and then saw this. I know nothing about JavaScript, but what kind of improvements are needed to make it on par with the latest diep? I Already program, and would gladly try to get it working again, if you would tell me what is needed. I obviously do not need the code as you put it up for download! XD

Thanks anyway if you do not reply,
oneohone

**Comment by terjanq** [comment]
You have to find those parts of source code where the changes on core are detected and remove them.

**Comment by Windfall-Github** [comment]
I think i'll try!

**Comment by Windfall-Github** [comment]
Hey @terjanq 

I found that if i remove the link to the main code in head.js, the game loads normally! I'm making progress! So it does NOT scan for scripts running. I am first going to make a kill counter work, and work from there. But diep.io is being updated CONSTANTLY!

Thanks, oneohone

**Comment by terjanq** [comment]
Yeah its always a new code because it's generated by "compiler" and it changes hourly. The only way to modify core.js is to make some regex pattern and change the  code before it's ran by browser.

**Comment by Windfall-Github** [comment]
And that will be a pain. Although, I have been working  on an automatic
"Cheat Engine" like script that auto-hacks. And i have succeeded in making
an insta-grow hack in slither io at  about 10 more points a second... But
diep is more fun...
